molecule-runtime/src/commonMain/kotlin/app/cash/molecule/molecule.kt

```diff
@@ -101,12 +101,14 @@ private fun <T> immediateClockFlow(body: @Composable () -> T): Flow<T> = flow {
  * to produce a [StateFlow] stream of [T] values.
  */
 public fun <T> CoroutineScope.launchMolecule(
+  context: CoroutineContext = EmptyCoroutineContext,
   mode: RecompositionMode,
   body: @Composable () -> T,
 ): StateFlow<T> {
```

Comment:
This and the function below need to be proper overloads rather than just adding a new parameter. We already shipped 1.0 so this would break binary compatibility.

I'll add the binary compatibility ~validator~ dumper plugin tomorrow.

Reply:
Good call, I would have seen this if I had the tests set up from the get-go. Let me know if adding telescoping [here](https://github.com/cashapp/molecule/pull/312/commits/b5092be63556903e67dc6cc0e8438e56d2aae7a8) addressed the binary compatibility.